lib/src/util/windows/process.cc

```diff
@@ -24,10 +24,7 @@ namespace facter { namespace util { namespace windows { namespace process {
     {
         HANDLE temp_token;
         if (!OpenProcessToken(GetCurrentProcess(), TOKEN_QUERY, &temp_token)) {
-            // pre-Vista will return ERROR_NO_SUCH_PRIVILEGE
```

Comment:
It's related because I'm repeating the OpenProcessToken call.

I removed it because it came from https://github.com/puppetlabs/puppet/blob/master/lib/puppet/util/windows/process.rb#L200, and reviewing the APIs further I don't see any place that error should be generated.

Actually, if anywhere it should be on the call to GetTokenInformation (where TokenElevation may not exist on 2003). This code is never called on 2003 because it's guarded by supports_elevated_security, so having the debug message shouldn't hurt.